### PR TITLE
Added a fix for master branches.

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Added a fix for master branches.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py [hunk 1]
```diff
@@ -55,8 +55,9 @@ def get_changes_in_pr(data):
     master_branch = subprocess.run(command, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     devel_branches = ""
     for branch in master_branch.stdout.split("\n"):
-        if "devel" in branch:
+        if "devel" in branch or "master" in branch or "main" in branch:
```
Comment (maxzieba) [inline ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py:58]: Commented on slack but anyways would be better to if branch in ["devel", "master", "main"]

Comment (bdenoun) [reply ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py:58]: agreed

**Review by bdenoun — APPROVED** [review]

File: ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py [hunk 2]
```diff
@@ -55,9 +55,12 @@ def get_changes_in_pr(data):
     master_branch = subprocess.run(command, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     devel_branches = ""
     for branch in master_branch.stdout.split("\n"):
-        if "devel" in branch:
+        if any(entry in branch for entry in ["devel","master","main"]):
             devel_branches = branch.strip()
-
+            break
+    if devel_branches == "":
+        print("Could not find the master branch: checks for devel, master, main")
```
Comment (maxzieba) [inline ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py:62]: If you're adding it like this, then I'd create a variable with (example name) accepted_as_devel_branches = ["devel","master","main"] as its easier to maintain later and reuse for prints/logs.

Comment (BenStarmerSmith) [reply ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py:62]: Thats a good point, will add.

Comment (BenStarmerSmith) [reply ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py]: Thats a good point, added.

**Pull request merged**